plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotIntegrationConnectorSmokeTest.java

```diff
@@ -833,6 +978,40 @@ public static Object of(
         }
     }
 
+    @Override
+    @Test
+    public void testShowCreateTable()
+    {
+        assertQueryFails("SHOW CREATE TABLE orders", ".* Table 'pinot.default.orders' does not exist$");
+    }
+
+    @Test
+    @Override // Pinot has a timeseries column
+    public void testSelectInformationSchemaColumns()
+    {
+        assertThat(query(format("SELECT column_name FROM information_schema.columns WHERE table_schema = '%s' AND table_name = 'region'", getSession().getSchema().orElseThrow())))
+                .skippingTypesCheck()
+                .matches("VALUES 'regionkey', 'name', 'comment', 'updated_at_seconds'");
+    }
```

Comment:
No need to call `getSession().getSchema()` since we already know the schema name in this class.
```java
    @Override
    public void testSelectInformationSchemaColumns()
    {
        // Override because there's updated_at_seconds column
        assertThat(query("SELECT column_name FROM information_schema.columns WHERE table_schema = 'default' AND table_name = 'region'"))
                .skippingTypesCheck()
                .matches("VALUES 'regionkey', 'name', 'comment', 'updated_at_seconds'");
    }
```